appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
```

Comment:
I assume you don't care about lazy-loading azure packages here because you already lazy-load "vscode-azureappservice" itself?

Reply:
That's correct. It certainly wouldn't hurt to apply lazy-loading in here, but I'll leave that for later.